lib/gst/player/gstplayer.c

```diff
@@ -2118,6 +2170,18 @@ subtitle_tags_changed_cb (GstElement * playbin, gint stream_index,
       GST_TYPE_PLAYER_SUBTITLE_INFO);
 }
 
+static void
+volume_notify_cb (GObject *obj, GParamSpec *pspec, GstPlayer *self)
+{
+  emit_volume_changed (self);
```

Comment:
Just inline those functions here, you're only ever going to call them from one place anyway :)